docs/tutorial/new_controller.rst

```diff
@@ -2,86 +2,72 @@
-	
-Name your controller and make pm.Controller its base class.
-Create public_settings like before in your model.
+    :end-before: #class begin
+    :lineno-match:
+
+Name your controller and make :py:class:`~pymoskito.simulation_modules.Controller` its base class.
+Create :py:data:`public_settings` like before in your model.
 Its entries will be accessable in the graphical interface of PyMoskito during runtime:
 
 .. literalinclude:: minimalSystem/controller.py
-	:start-after: #class begin
-	:end-before: #init
-	:lineno-match:
+    :start-after: #class begin
+    :end-before: #init
+    :lineno-match:
 
 Within the constructor, it is obligatory to set the input order and an input type. 
 The input order determines, how many derivatives of the trajectory will be required.
 Valid input types are *system_state*, *system_output*, *Observer* and *Sensor*.
-After all necessary updates, call the constructor of the base class:
+After all necessary updates, call the constructor of the base class as seen in line :py:data:`20`.
 
-.. literalinclude:: minimalSystem/controller.py
-	:start-after: #init
-	:end-before: #matrices
-	:lineno-match:
-
-Store the linearized system matrices and the equilibrium state. 
-Additionally, initilize an output variable.
-To make matrix operations possible, use the array type provided by NumPy:
-
-.. literalinclude:: minimalSystem/controller.py
-	:start-after: #matrices
-	:end-before: #calculations
-	:lineno-match:
+Store the linearized system matrices and the equilibrium state.
+To make matrix operations possible, use the array type provided by NumPy.
 
 PyMoskito's :doc:`Controltools <../modules/controltools>` provide functions
 to calculate the values of a linear state feedback and a prefilter:
 
 .. literalinclude:: minimalSystem/controller.py
-	:start-after: #calculations
-	:end-before: #control
-	:lineno-match:
-
-The only method to implement contains the control law.
+    :start-after: #init
+    :end-before: #control
+    :lineno-match:
+
+The only method to implement contains the control law and will be called by the solver during runtime.
+Its parameters are the current time, the current values of trajectory,
+feedforward and controller input.
+The parameter :py:data:`**kwargs` is free to be used as needed, 
+in this case it is ignored.
 Since the controller stabilizes the system in [0,0,0,0],
-the subtraction of the equilibrium state is necessary
-:
-	
+the subtraction of the equilibrium from the state vector is necessary:
+
 .. literalinclude:: minimalSystem/controller.py
-	:start-after: #control
-	:end-before: #register
-	:lineno-match:
-
-In case you would like to implement more than one controller, 
-you can do so within the same file.
-	
-Do not forget to link all your controllers to the toolbox at the 
-very bottom of your code:
-	
+    :start-after: #control
+    :end-before: #register
+    :lineno-match:
+
+Do not forget to link your controller to the toolbox at the 
```

Comment:
Maybe link is a little bit misleading here. It's more like a registration process that tells the toolbox that this module is available.
However as your comment above already mentions this instruction should be moved to the section `Starting the Program`